docs/macros-attributes/default.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "#[ink(default)]"
+slug: /macros-attributes/default
+hide_title: true
+---
+
+//todo: change image
```

Comment:
I think we need a new image for this page. Who could prepare it ?

Reply:
Yep. I have notified our illustrator to come up with an image. I send a link here once it's ready

Reply:
@kziemianek here is the svg: 
![# ink(default)](https://user-images.githubusercontent.com/17993104/234345037-a52baba8-d0d0-44d7-9bc8-75463c9d4432.svg)

Reply:
@SkymanOne thank you, I've added commit with image 🙃